lib/doc/doxygen/snippets/CTestTestfile.cmake

```diff
@@ -0,0 +1,6 @@
+# CMake generated Testfile for 
```

Comment:
Also this file has been created by cmake...

Reply:
Removed.